src/services/books.js

```diff
@@ -0,0 +1,33 @@
+const BASE_URL = process.env.BASE_URL || 'http://localhost:4000'
+
+const booksService = {}
+
+booksService.getAll = function () {
```

Comment:
Later on, with the authentication part, you may need to supply an Authorization header that has a token (depending on how you authorize requests on your backend). So I highly recommend the use of `axios` to streamline this.